packages/aragon-wrapper/src/index.js

```diff
@@ -200,53 +202,56 @@ export default class Aragon {
     // TODO: Only includes apps in the namespace `keccak256("app")`
     // TODO: Refactor this a bit because it's pretty much an eye sore
     this.identifiers = new Subject()
-    this.appsWithoutIdentifiers = this.permissions
-      .map(Object.keys)
-      .map((addresses) =>
+    this.appsWithoutIdentifiers = this.permissions.pipe(
+      map(Object.keys),
+      map((addresses) =>
         addresses.filter((address) => !addressesEqual(address, this.kernelProxy.address))
-      )
-      .switchMap(
+      ),
+      switchMap(
         (appAddresses) => Promise.all(
           appAddresses.map((app) => this.getAppProxyValues(app))
         )
-      )
-      .map(
+      ),
+      map(
         (appMetadata) => appMetadata.filter((app) => this.isApp(app))
-      )
-      .flatMap(
+      ),
+      flatMap(
         (apps) => Promise.all(
           apps.map(async (app) => Object.assign(
             app,
             await this.apm.getLatestVersionForContract(app.appId, app.codeAddress)
               .catch(() => ({}))
           ))
         )
-      )
+      ),
       // Replaying the last emitted value is necessary for this.apps' combineLatest to not rerun
       // this entire operator chain on identifier emits (doing so causes unnecessary apm fetches)
-      .publishReplay(1)
+      publishReplay(1)
+    )
     this.appsWithoutIdentifiers.connect()
 
-    this.apps = this.appsWithoutIdentifiers
-      .combineLatest(
-        this.identifiers.scan(
-          (identifiers, { address, identifier }) =>
-            Object.assign(identifiers, { [address]: identifier }),
-          {}
-        ).startWith({}),
-        function attachIdentifiers (apps, identifiers) {
-          return apps.map(
-            (app) => {
-              if (identifiers[app.proxyAddress]) {
-                return Object.assign(app, { identifier: identifiers[app.proxyAddress] })
-              }
-
-              return app
+    this.apps = combineLatest(
+      this.appsWithoutIdentifiers,
+      this.identifiers.pipe(scan(
+        (identifiers, { address, identifier }) =>
+          Object.assign(identifiers, { [address]: identifier }),
+        {}
+      )),
+      startWith({}),
+      function attachIdentifiers (apps, identifiers) {
```

Comment:
It looks like the [result selectors are being deprecated with v6](https://rxjs-dev.firebaseapp.com/guide/v6/migration#result-selectors-removed-or-deprecated), what do you think about migrating them over (it looks like it's usually just a pipe to `map` instead)?

Reply:
Turns out `apps` is an array in this context